xonsh/prompt_toolkit_key_bindings.py

```diff
@@ -32,3 +32,11 @@ def _(event):
         indent instead of autocompleting.
         """
         event.cli.current_buffer.insert_text(env.get('INDENT'))
+
+    @handle(Keys.BackTab)
+    def _(insert_literal_tab):
```

Comment:
Hi @gforsyth - I think I might not have been clear enough before. `event` is fine.  It is the method name (`_`) that should be expanded to something more meaningful. Thanks!